README.md

```diff
@@ -201,140 +185,13 @@ The `access_list` is the series of entries defining how to authorize claims.
 In the above example, the plugin authorizes access for the holders of "roles"
 claim where values are any of the following: "anonymous", "guest", "admin".
 
-## Plugin Developers
-
-This section of the documentation targets a plugin developer who wants to issue
-JWT tokens as part of their plugin.
-
-Please see [caddy-auth-portal](https://github.com/greenpau/caddy-auth-portal/blob/0bc10a3de90f63d44a6617ccbd284c2d23f73e39/pkg/backends/local/backend.go#L26)
-for an example how to issue JWT tokens.
-
-First, a developer would need to create `TokenProviderConfig` object via
-`NewTokenProviderConfig()`.
-
-```
-tokenProvider := jwt.NewTokenProviderConfig()
-```
-
-Second, set the `TokenProviderConfig`
-[properties](https://github.com/greenpau/caddy-auth-portal/blob/0bc10a3de90f63d44a6617ccbd284c2d23f73e39/pkg/backends/local/backend.go#L274-L297), e.g.:
-
-* `TokenName`
-* `TokenIssuer`
-* `TokenOrigin`
-* `TokenLifetime`
-
-Next, create a claim:
-
-```go
-    claims := &jwt.UserClaims{}
-    claims.Subject = username
-    claims.Email = username
-    claims.Name = "Smith, John"
-    claims.Roles = append(claims.Roles, "anonymous")
-    claims.Roles = append(claims.Roles, "guest")
-    claims.Origin = tokenProvider.TokenOrigin
-    claims.ExpiresAt = time.Now().Add(time.Duration(tokenProvider.TokenLifetime) * time.Second).Unix()
-```
-
-Finally, having created claims, the developer can create a token string:
-
-```go
-userToken, err := claims.GetToken("HS512", []byte(m.TokenProvider.TokenSecret))
-```
-
-## Role-based Access Control and Access Lists
-
-### Sources of Role Information
-
-By default, the plugin finds role information in `roles` key of a token payload.
-In the below example, the use has a single role, i.e. `anonymous`.
-
-```json
-{
-  "exp": 1596031874,
-  "sub": "jsmith",
-  "name": "Smith, John",
-  "email": "jsmith@gmail.com",
-  "roles": [
-    "anonymous"
-  ],
-  "origin": "localhost"
-}
-```
-
-Additionally, the token validation component of the plugin recognized that roles
-may be in other parts of a token, e.g. `app_metadata - authorization - roles`:
-
-```json
-{
-  "app_metadata": {
-    "authorization": {
-      "roles": ["admin", "editor"]
-    }
-  }
-}
-```
-
-References:
-
-* [Auth0 Docs - App Metadata](https://auth0.com/docs/users/concepts/overview-user-metadata)
-* [Netlify - Role-based access control with JWT - External providers](https://docs.netlify.com/visitor-access/role-based-access-control/#external-providers)
-
-### Anonymous Role
-
-By default, if the plugin does not find role information in JWT token, then
-automatically treats the token having the following two roles:
-
-* `anonymous`
-* `guest`
+### Access List
```

Comment:
@njones , please match current `README.md`. I also think you should add documentation related to your changes.